examples_tests/13.BRDF_Explorer/CMakeLists.txt

```diff
@@ -4,8 +4,14 @@ if(NOT RES)
 	message(FATAL_ERROR "common.cmake not found. Should be in {repo_root}/cmake directory")
 endif()
 
+set(BRDF_SOURCES
```

Comment:
rename the BRDF prefix to BRDF_EXPLORER, I don't want it colliding with actual pure-BRDF stuff later on or to be misunderstand

Reply:
updated.